ui/src/app/views/dataset-page/component.tsx

```diff
@@ -89,15 +90,33 @@ export const ValueListItem: React.FunctionComponent <
     </div>;
   };
 
-export const DataSetPage = withCurrentWorkspace()(class extends React.Component<
-  {workspace: WorkspaceData},
-  {creatingConceptSet: boolean, conceptDomainList: DomainInfo[],
-    conceptSetList: ConceptSet[], cohortList: Cohort[], loadingResources: boolean,
-    confirmDeleting: boolean, editing: boolean, resource: RecentResource,
-    rType: ResourceType, selectedConceptSetIds: number[], selectedCohortIds: number[],
-    valueSets: ValueSet[], selectedValues: DomainValuePair[], openSaveModal: boolean,
-    queries: Array<DataSetQuery>, includesAllParticipants: boolean
-  }> {
+interface Props {
+  workspace: WorkspaceData;
+}
+
+interface State {
```

Comment:
Becoming more and more sold on the idea of interfaces. This is way more readable.